tests/instrumentation/perf_tests/tests.py

```diff
@@ -0,0 +1,47 @@
+import mock
+import pytest
+
+import elasticapm
+from tests.instrumentation.perf_tests.perf_util import go_someplace_else
+
+
+@pytest.mark.parametrize('elasticapm_client', [{
+    'include_paths': ('tests.instrumentation.perf_tests', 'instrumentation.perf_tests', 'a.b', 'c.d.e', 'f.g.h.x'),
+    'exclude_paths': ('tests.instrumentation.django_tests', 'elasticapm')
+}], indirect=True)
+def test_stacktrace_performance(elasticapm_client, benchmark):
+    with mock.patch('elasticapm.traces.TransactionsStore.should_collect') as should_collect:
+        def do_it():
+            elasticapm_client.begin_transaction('perf')
+            go()
```

Comment:
Maybe you should have a small (parameterised) backoff time period here to get a more realistic impression of overhead?